src/sidebar/panel.js

```diff
@@ -122,7 +122,9 @@ ClassicEditor.create(document.querySelector('#editor'), {
               setAnimation(true); // animateSyncIcon, syncingLayout, warning
               syncingInProcess = true;
             }
-            if (! waitingToReconnect) {
+            if (! waitingToReconnect && isAuthenticated) {
```

Comment:
I guess you can have a if for the `!waitingToReconnect` and another one for the `isAuthenticated` condition.

Reply:
Oh, I just saw this now. Well thanks for changing :smiley: